src/hooks/primary/useGetPrimaryNameTransactionFlowItem/index.ts

```diff
@@ -101,19 +110,19 @@ export const useGetPrimaryNameTransactionFlowItem = (
           ? {
               resumeable: true,
               intro: {
-                title: [getIntroTranslation(introType, 'title'), { ns: 'transactionFlow' }],
+                title: [getIntroTranslation(introType, 'title'), { ns: 'transactionFlow' }] as any,
```

Comment:
is as any required here

Reply:
Nope. Thanks for the catch. I forgot to delete it.